api/pom.xml

```diff
@@ -63,16 +63,21 @@
 
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
                 <configuration>
                     <archive>
                         <manifest>
                             <mainClass>org.jboss.marshalling.Version</mainClass>
                         </manifest>
+                        <manifestEntries>
+                            <Automatic-Module-Name>org.jboss.marshalling.api</Automatic-Module-Name>
```

Comment:
To be consistent with the artifact name, I think the module name for the API module should be `org.jboss.marshalling`.